metrics/storage/web-tooling_dockerfile/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Set up an Ubuntu image with 'web tooling' installed
+
+# Usage: FROM [image name]
+FROM ubuntu:18.04
+
+# Version of the Dockerfile
+LABEL DOCKERFILE_VERSION="1.0"
+
+# URL for web tooling test
+ENV WEB_TOOLING_URL "https://github.com/v8/web-tooling-benchmark"
+ENV NODEJS_URL "https://deb.nodesource.com/setup_14.x"
+
+RUN apt-get update && \
+	apt-get install -y build-essential git curl sudo && \
+	apt-get remove -y unattended-upgrades && \
+	curl -OkL https://deb.nodesource.com/setup_14.x && chmod +x setup_14.x && ./setup_14.x && \
```

Comment:
There are lots of instances of `setup_14.x` - could you use variables to avoid this?